api/controllers/v1/entrance/find.js

```diff
@@ -21,9 +21,8 @@ module.exports = async (req, res) => {
       .populate('names');
 
     const params = { searchedItem: `Entrance of id ${req.params.id}` };
-    // TODO How to delete/restore entity ?
-    if (!entrance || entrance.isDeleted)
-      return res.notFound(`${params.searchedItem} not found`);
+
+    if (!entrance) return res.notFound(`${params.searchedItem} not found`);
 
```

Comment:
Is it needed to populate all child entities when the entrance is deleted ?
May be a early return could be added ?
```suggestion
if(entrance.isDeleted){
    return ControllerService.treatAndConvert(
      req,
      null,
      entrance,
      params,
      res,
      toEntrance
    );
}
```

Reply:
Done !